conf/application.conf

```diff
@@ -457,6 +457,7 @@ schema = """
 					"type": "string",
 					"minLength": 1,
 					"maxLength": 8,
+					"pattern": "^(\\d){1,8}$",
```

Comment:
this is for NSI , 
similarly do we have same regex for SF in our help-to-save-api as well ?